server.js

```diff
@@ -128,11 +130,14 @@ if (config.hasAzure) {
 const upload = multer({
     storage: multer.memoryStorage(),
     limits: {
-        fieldSize: 1e7, // max in bytes
-        fileSize: 1e7, // max in bytes
-        parts: 100, // max number of fields + files
+        fieldSize: config.fileUploadMaxBytes,
+        fileSize: config.fileUploadMaxBytes,
+        parts: config.fileUploadMaxParts,
     },
 });
+config.fileUploadMaxBytesFormatted = filesize(config.fileUploadMaxBytes, {base: 10, round: 0});
+app.use(function(req, res, next) {res.locals.config.fileUploadMaxBytes = config.fileUploadMaxBytes; next();});
```

Comment:
I'm a little confused on how this works - down on L167, we assign `res.locals.config = config`. Could you double check that?

Reply:
Haha, many thanks! I completely failed to realized that we were already making `config` available as `locals.config`. I thought to myself “it would be very convenient if config was available inside locals” 😄 — for whatever reason I only made a few fields explicitly available.

I removed all this duplication.